modules/vaos/spec/request/v2/appointments_request_spec.rb

```diff
@@ -784,22 +786,20 @@ def stub_clinics
         end
 
         context 'when clinic and location_id are present' do
-          let(:updated_appointment) { { clinic: 'Test Clinic', location_id: 1 } }
-
           it 'updates the service name, physical location, friendly name, and location' do
             allow_any_instance_of(VAOS::V2::MobileFacilityService).to receive(:get_clinic)
               .and_return(service_name: 'Service Name', physical_location: 'Physical Location')
-            allow_any_instance_of(VAOS::V2::MobileFacilityService).to receive(:get_facility)
-              .and_return('Location')
-            allow_any_instance_of(described_class).to receive(:updated_appointment).and_return(updated_appointment)
-
-            put '/vaos/v2/appointments/70060', params: { status: 'cancelled' }, headers: inflection_header
+            stub_facilities
+            VCR.use_cassette('vaos/v2/appointments/cancel_appointments_200',
```

Comment:
Needed to update the test to use more realistic traffic and mock facility due to logging and post processing expectations.